src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -1,14 +1,34 @@
 package racingcar.domain
 
-class Car {
+data class Car(val name: String) {
 
-    val history = mutableListOf<CarAction>()
+    init {
+        require(name.length <= 5)
+    }
+
+    private val _history = mutableListOf<CarAction>()
+    val history
+        get() = _history.toList()
 
     fun move() {
```

Comment:
`Car` 외에 사용하는 곳이 없어 보이네요. public이 아닌 private으로 바꾸면 어떨까요?

```suggestion
    private fun move() {
```